STYLE_GUIDE.md

```diff
@@ -57,13 +57,13 @@
 
 ### Folders and files
 
-  * Kebab-Case for folders.
+  * `kebab-case` for folders.
 
     > Covers if folder will be extracted to its own package some day. [[+]](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#name)
 
-  * Camel case or Pascal case for files.
+  * `camelCase` or `PascalCase` for files.
 
-    > Has problems on renaming case-sensitive files with git.
+    > Has problems on [commiting case-sensitive filename changes with Git](https://stackoverflow.com/a/20907647).
```

Comment:
could you change the link to use `[[+]](linkUrl)` template which is using in the rest of file
```
> Has problems on commiting case-sensitive filename changes with Git [[+]](https://stackoverflow.com/a/20907647).
```